### ISSUE TITLE
Member is "unknown" but able to be jumped to

### ISSUE THREAD

**Issue by Gobot1234** [microsoft/pyright#3698]
**Describe the bug**
Members of a class are reported as being unknown when they should be known

**To Reproduce**
I'm not really sure, I've tried copying the source for ChatGroup into the files where its used and this causes the errors to go away.

**Expected behavior**
The member shouldn't be unknown.

**Screenshots or Code**
id64 and id are properties defined in a superclass of ChatGroup, https://github.com/Gobot1234/steam.py/blob/997621880145f41cbc12ffe3f065d17872bd1652/steam/abc.py#L127-135
```py
from .chat import ChatGroup

class Clan(ChatGroup[ClanMember, ClanChannel]):
    def foo(self) -> None:
        self.id64
```
![image](https://user-images.githubusercontent.com/50501825/178981391-e08a8017-ebb8-4832-95d6-887333709459.png)
this only happens in steam/clan.py if I use the exact same code outside of the library's directory, it works fine
![Screenshot 2022-07-14 at 13 27 22](https://user-images.githubusercontent.com/50501825/178982395-b57551d0-9611-43d5-96f7-f5f4a8b0c39b.png)

Other weird tid-bits
Here (https://github.com/Gobot1234/steam.py/blob/997621880145f41cbc12ffe3f065d17872bd1652/steam/state.py#L950) clan.id is marked as unknown, clan is of type Clan so I would have thought this also would have been erroneously warned about like with group.id (Group is also a subclass of ChatGroup)
![image](https://user-images.githubusercontent.com/50501825/178983689-d751a696-ca30-4a96-a159-a7c953c458ee.png)

**VS Code extension or command-line**
pylance: 2022.7.30

**Comment by erictraut** [comment]
I don't have enough context to figure out what's going on here. Could you provide more details about how I can repro the problem? Which libraries (and versions) are involved? You mentioned the class `ChatGroup`. It looks like that's part of this `steam.py` library that you're developing? Do I need to install the library to repro the problem? Or do I need to clone the library code?

**Comment by Gobot1234** [comment]
I think I just didn't wait long enough for the types to resolve because I now can't reproduce the problem with the same bit of code.
